npm/azure-npm.yaml

```diff
@@ -148,18 +148,19 @@ metadata:
 data:
   azure-npm.json: |
     {
-        "ResyncPeriodInMinutes": 15,
-        "ListeningPort":         10091,
-        "ListeningAddress":      "0.0.0.0",
-        "ApplyMaxBatches": 100,
+        "ResyncPeriodInMinutes":       15,
```

Comment:
Add windows ds to this yaml.

Reply:
discussed offline that there may be script dependencies on either yaml